config/default.yml

```diff
@@ -148,6 +148,71 @@ Style/BarePercentLiterals:
     - percent_q
     - bare_percent
 
+Style/Blocks:
+  EnforcedStyle: multiline
+  SupportedStyles:
+    # The `multiline` style enforces braces around single line blocks and
+    # do..end around multi-line blocks.
+    - multiline
```

Comment:
Or `line_lenght_based`? I'm not sure - I just know I don't like the current name.

I'd also suggest renaming `weirich` to `semantic` - that's how he referred to this style himself.